ecs-agent/csiclient/csi_client.go

```diff
@@ -60,6 +62,13 @@ func NewCSIClient(socketIn string) csiClient {
 	return csiClient{csiSocket: socketIn}
 }
 
+// Returns a CSI client configured with default settings.
+// The default socket filepath is "/var/run/ecs/ebs-csi-driver/csi-driver.sock".
+func NewDefaultCSIClient() CSIClient {
```

Comment:
Q: Will this be used in a future PR? It might be worth adding a comment of when to use this will be used.

Reply:
Synced offline. Thanks!

Reply:
Updated the description! Thanks!